<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeetingRoom</class>
 <widget class="QWidget" name="MeetingRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>450</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(12, 12, 12);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="CB_micSet">
           <property name="text">
            <string>麦克风</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CB_cremaSet">
           <property name="text">
            <string>摄像头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CB_creenSet">
           <property name="text">
            <string>共享</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser_msg">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_msg"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_msgSend">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget_members"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_invite"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_invite">
           <property name="text">
            <string>邀请</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
